packages/api/lambdas/payload-logger.js

```diff
@@ -28,7 +28,7 @@ async function kinesisEventLogger(event, logger = log) {
  * @returns {void} returns nothing
  */
 async function handler(event) {
-  return kinesisEventLogger(event);
+  return await kinesisEventLogger(event);
```

Comment:
We don't need to `await` this since it's not async, right? And it doesn't return a promise, `kinesisEventLogger` returns an object.

Reply:
Yeah.. This one was a bit of a compromise. `async` is not strictly necessary here, but not having it will necessitate changing the [lambda handler from async-style to callback-style](https://docs.aws.amazon.com/lambda/latest/dg/nodejs-handler.html) (`function handler(event, context, callback) {`. At least in my testing, I do get the expected return value from `await`ing a non-async function. If you think it should be fully converted to callback style I can do that.

Reply:
That compromise seems fine to me. Thanks for explaining!